app/(white_navbar)/explore/RetroPGF5/[name]/_component/_Project/OverviewSection.tsx

```diff
@@ -0,0 +1,30 @@
+import { handleCategoryRound5 } from '../../../../../../lib/InitialMount'
+import { iRetroPGF5Project } from '../../../RetroType5'
+
+interface OverviewSectionProps {
```

Comment:
This should name `iOverviewSection` like interface in other components

Reply:
That's right!, Thank for flagging this.